### PR TITLE
add test for streamable -> json conversion

### PR DESCRIPTION
**Pull request by arvidn**
I am about to propose a refactor to `recurse_jsonify()`, so I would like to have tests first

### CODE REVIEW

**Review by trepca — DISMISSED** [review]
nesting would be nice to have too (A.b => B.c => C)

**Comment by arvidn** [comment]
this test covers one level on nesting. I wouldn't expect testing three layers would add that much coverage:
https://github.com/Chia-Network/chia-blockchain/pull/11527/files#diff-8e5b980067a2cb9e5545b0d7933776df9dd1412fdabe53bc8b0c075065f49689R93

**Comment by trepca** [comment]
> this test covers one level on nesting. I wouldn't expect testing three layers would add that much coverage: https://github.com/Chia-Network/chia-blockchain/pull/11527/files#diff-8e5b980067a2cb9e5545b0d7933776df9dd1412fdabe53bc8b0c075065f49689R93

I meant nested with a custom class, but I guess custom classes are not much different from Tuple

**Comment by arvidn** [comment]
There the `Inner` class which is nested inside the `NestedTest` class

File: tests/core/util/test_jsonify.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                24,
+                25,
+                26,
+                27,
+                28,
+                29,
+                30,
+                31,
+            ]
+        ),
```
Comment (altendky) [inline tests/core/util/test_jsonify.py:59]: This way I don't have to read every line to know what is happening.

```suggestion
        bytes32(range(32)),
```

**Comment by altendky** [comment]
I think adding a direct nesting would be good as well.  Maybe also nesting through a list.  It may or may not be 'relevant' with the present implementation, but it seems reasonable that there might be differences in some people's implementations that may arise in the future, if not now.

File: tests/core/util/test_jsonify.py (hunk 1 above)
Comment (arvidn) [reply tests/core/util/test_jsonify.py:59]: heh, yeah. it didn't look like this before `black` btw :)

**Review by altendky — APPROVED** [review]
I appreciate the separate test cases, as opposed to just mixing all the nesting types in one test.

**Pull request merged**